app/routes.php

```diff
@@ -79,8 +79,11 @@
 
         // trailing slash below added by mistake, ? added later to make it optional.
         // todo make FE stop sending the slash and then remove here.
-        $versionGroup->post('/emailVerificationToken/?', EmailVerificationToken\Create::class)
```

Comment:
apparently this ? thing doesn't work here. New bug from adding the ? wasn't deployed to any env.